docs/content/en/contribute/general/_index.md

```diff
@@ -4,10 +4,12 @@ description: Information relevant to all contributions
 weight: 100
 ---
 
-This section is under development.
-Contents are not complete,
-but all material here has been reviewed for technical accuracy.
-
 Keptn software and documentation
 is developed and maintained by the Keptn community.
 We invite you to join us.
+
+## Understanding related technologies
+
+## Guidelines for docs contributing
+
+## Working with github
```

Comment:
I'm not sure why you have these headers here.  This is the landing page for the "General" section of the Contributing Guide.  All the subsections will be listed here automatically.  You could do a little bullet list of the sections with a brief description of the contents to make sort of a "happy path" for the writer -- that's up to you to decide if that is useful.